app/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
     <uses-permission android:name="com.android.providers.tv.permission.WRITE_EPG_DATA" /> <!-- Used for Android TV watch next -->
     <uses-permission android:name="android.permission.UPDATE_PACKAGES_WITHOUT_USER_ACTION" /> <!-- Used for updates without prompt -->
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" /> <!-- Used for update service -->
-
+    <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS" />
```

Comment:
You request REQUEST_IGNORE_BATTERY_OPTIMIZATIONS without using  ACTION_REQUEST_IGNORE_BATTERY_OPTIMIZATIONS

Reply:
Look at https://stackoverflow.com/questions/44862176/request-ignore-battery-optimizations-how-to-do-it-right